sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuMapInPandasExec.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Do we need to move this file into a directory of shims, since I saw the shims json lines are added?